### PR TITLE
fix(util): added offset to the smooth-scroll util

### PR DESCRIPTION
**Pull request by RaphaelAmadeu**
### Related Ticket(s)
#1620 

### Description

An offset to the scrolling was added, so the entire heading appears properly after scrolling on mobile devices.

### Changelog

**New**

- Added scroll offset

### CODE REVIEW

File: packages/utilities/src/utilities/smoothScroll/smoothScroll.js [hunk 1]
```diff
@@ -31,9 +31,13 @@ const smoothScroll = (e, selector) => {
   } else {
     return null;
   }
-  document.querySelector(getSelector).scrollIntoView({
+
+  window.scroll({
+    top:
+      document.querySelector(getSelector).getBoundingClientRect().top -
+      50 +
```
Comment (jeffchew) [inline packages/utilities/src/utilities/smoothScroll/smoothScroll.js]: I feel like this should be an optional parameter, where the default is 0. Thoughts?

Comment (RaphaelAmadeu) [reply packages/utilities/src/utilities/smoothScroll/smoothScroll.js]: This is a nice idea, i've implemented it on my latest commit

File: packages/utilities/src/utilities/smoothScroll/smoothScroll.js [hunk 2]
```diff
@@ -31,9 +32,13 @@ const smoothScroll = (e, selector) => {
   } else {
     return null;
   }
-  document.querySelector(getSelector).scrollIntoView({
+
+  window.scroll({
```
Comment (jeffchew) [inline packages/utilities/src/utilities/smoothScroll/smoothScroll.js]: @RaphaelAmadeu are we switching from `scrollIntoView`?

Comment (RaphaelAmadeu) [reply packages/utilities/src/utilities/smoothScroll/smoothScroll.js]: @jeffchew ,
`scrollIntoView` does not support offset, and it was causing the heading to not be shown completely on mobile devices.

**Pull request merged**